app/src/main/java/com/duckduckgo/app/notification/NotificationHandlerService.kt

```diff
@@ -75,107 +42,21 @@ class NotificationHandlerService : IntentService("NotificationHandlerService") {
 
     @VisibleForTesting
     public override fun onHandleIntent(intent: Intent?) {
-        val pixelSuffix = intent?.getStringExtra(PIXEL_SUFFIX_EXTRA) ?: return
-
-        when (intent.type) {
-            APP_LAUNCH -> onAppLaunched(pixelSuffix)
-            CLEAR_DATA_LAUNCH -> onClearDataLaunched(pixelSuffix)
-            CANCEL -> onCancelled(pixelSuffix)
-            WEBSITE -> onWebsiteNotification(intent, pixelSuffix)
-            CHANGE_ICON_FEATURE -> onCustomizeIconLaunched(pixelSuffix)
-            else -> {
-                schedulableNotificationPluginPoint.getPlugins().forEach {
-                    when (intent.type) {
-                        it.getSchedulableNotification().launchIntent -> {
-                            it.onNotificationLaunched()
-                        }
-                        it.getSchedulableNotification().cancelIntent -> {
-                            it.onNotificationCancelled()
-                        }
-                    }
-                }
-            }
-        }
-
-        if (intent.getBooleanExtra(NOTIFICATION_AUTO_CANCEL, true)) {
-            val notificationId = intent.getIntExtra(NOTIFICATION_SYSTEM_ID_EXTRA, 0)
-            clearNotification(notificationId)
+        if (intent == null) return
```

Comment:
This handles now only the action of cancelling a notification. In the future this class will be removed entirely.